src/test/java/com/mongodb/jdbc/MongoDatabaseMetaDataTest.java

```diff
@@ -363,30 +405,37 @@ void testGetFunctions() throws SQLException {
         rs = databaseMetaData.getFunctions(null, null, "%s%");
         assertEquals(countRows(rs), 0);
     }
+}
+
```

Comment:
Does gradle correctly run these when they are in the same file? I was just using separate files to be safe.